packages/modular-scripts/src/test/index.ts

```diff
@@ -85,14 +87,28 @@ async function test(
   // create argv from jest Options
   const cleanArgv: string[] = [];
 
-  // pass in path to configuration file
+  // pass in jest configuration
   const { createJestConfig } = await import('./config');
-  cleanArgv.push(
-    '--config',
-    `"${JSON.stringify(
+
+  // Only used when running on Windows
+  const tempConfigPath = path.join(tmp.dirSync().name, 'temp-jest-config.json');
+
+  // Windows: pass in configuration as a file as jest won't accept "" around the object and Windows doesn't accept JSON object {} passed directly
```

Comment:
We now have two copies of this code with the Windows temp hack; can we factor it out in a `invokeJest` function?

Reply:
Yep good point, will do

Reply:
Done